### DISCUSSION TITLE
Welcome to Tilde Discussions!

### DISCUSSION THREAD

**Discussion by sheenarbw** [Umuzi-org/Tilde#487] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we learn and
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kenneth-bit** [comment]
Mpumele Mfokeng here, I am a Recruit Data Scientist, Coming To You Live From The East Rand. 💯😎

**Comment by UnityGameDesign-max** [comment]
Tumelo Mahlangu here. I was a web developer recruit last year😎. I really want to join the community and build amazing cool stuff and learn more 💪.
